packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -284,14 +285,16 @@ const makeRenderer = (
 				store.dispatch({ type: SERVER_RENDER, payload: url });
 
 				if (checkReady(store.getState())) {
-					addFlags(store).then(() => {
+					const memberObj = (store.getState().api.self || {}).value || {};
```

Comment:
First time looking at this code, but can't you just define memberObj once on line 286?